### PR TITLE
new desciption

### PR DESCRIPTION
**Pull request by teseoch**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by teseoch** [comment]
@conda-forge-admin, please rerender

**Comment by teseoch** [comment]
@conda-forge-admin, please rerender

**Comment by teseoch** [comment]
@conda-forge/core issue with re-rendering

**Comment by ocefpaf** [comment]
> @conda-forge/core issue with re-rendering

The webservice may take a while.

**Comment by teseoch** [comment]
@conda-forge-admin, please rerender

**Comment by teseoch** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -27,11 +27,11 @@ requirements:
     - libblas
     - libcblas
     - numpy
-    - suitesparse
+    - suitesparse  # [not linux_ppc64le and not linux_aarch64]
```
Comment (jjhelmus) [inline recipe/meta.yaml]: Try `# [not ppc64le and not aarch64]`

Comment (teseoch) [reply recipe/meta.yaml]: thanks!

**Comment by teseoch** [comment]
@conda-forge-admin, please rerender

**Pull request merged**